doc/ref_model/chapters/chapter10.md

```diff
@@ -40,8 +40,9 @@ Gaps
 
 This section addresses major open issues identified in the development of the Reference Model, Reference Architecture, Reference Implementation of the Common Cloud Infrastructure Lifecycle Framework. 
 
-## 10.3.1 Discovery [Initial language]
-The VNFs and NFVI frameworks should be able to discover each other and exchange their capabilities required or offered. CNTT reference model specifies a model, but OpenStack does not provide APIs to support such capabilities. Following should be supported: NFVI should be able to publish the capabilities it offers to VNFs/CNFs VNFs/CNFs should be able to query the NFVI for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but, should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
+## 10.3.1 Discovery
+The workloads (VNFs/CNFs) and Cloud Infrastructure should be able to discover each other and exchange their capabilities required or offered. One of the key pain points for most of the operators is the VNF/CNF onboarding - both in terms of time and complexity. It could take weeks or months to onboard a VNF/CNF. There are lots of static and laborious checks performed to ensure the compatibility of the workloads with the corresponding Cloud Infrastructure. 
+The onboarding of the workloads (network functions) should be as much automated as possible. The CNTT conformant workloads and Cloud Infrastructure should be able to discover and negotiate their capabilities. Following should be supported: Cloud Infrastructure should be able to publish the capabilities it offers to workloads (network functions) and workloads should be able to query the Cloud Infrastructure for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
 
 ## 10.3.2 Support Load Balance of VNF/CNFs [Initial language]
 Ability to load balance workflows through multiple instances of same VNF or CNF - e.g. using ECMP to distribute workloads through the multiple instances of Firewall. As an example imagine a distributed finance application with multiple instances of Web-tier and DB-tier. The traffic needs to flow through multiple instances of Firewall (for HA as well load balancing). There is no simple way to accomplish this in OpenStack.
```

Comment:
What about the OSTK LBaaS - Octavia?

Reply:
@pgoyal01 Thanks for your suggestions for the content for 10.3.1 (Discovery). I have have committed those updates. 

This PR is to update section 10.3.1 only (please see the title of this PR). This does not update 10.3.2 or 10.3.4. Do you want to address those as part of this PR or a separate one? 
Please advise.